src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -51,7 +51,7 @@ class DominionCvrReader extends BaseCvrReader {
   private Map<Integer, String> precinctPortions;
   // map of contest ID to Contest data
   private Map<String, Contest> contests;
-  private List<Candidate> candidates;
+  private HashMap<String, Candidate> candidates;
```

Comment:
same as another comment: is it possible to signal what this string is in the HashMap? Maybe by changing the variable to candidateCodeToCandidate or candidatesByCode or something like that.